### ISSUE TITLE
Is `echo` really needed?

### ISSUE THREAD

**Issue by Enjoyzz** [tedious/JShrink#121]
Is `echo` really needed here? there is already a concatenation `$this->output.= $char;`, and so buffering is removed, and now there is output and writing to the variable

https://github.com/tedious/JShrink/blob/8dd2fd54293e94fa1536c3c66700c707d814e043/src/JShrink/Minifier.php#L208

**Comment by tedivm** [comment]
Good call! I've got a PR up now to fix this- once tests pass I'll issue a bugfix release.

**Comment by tedivm** [comment]
Version v1.6.2 and higher will no longer echo- thanks for reporting!
